### DISCUSSION TITLE
【猴子比你還會】馬斯克推出腦波控制器，讓猴子用「念力」玩電玩

### DISCUSSION THREAD

**Discussion by yi-1111** [1101-GS4719-C/Python-Community-GS4719-C#503] [科技新知]
馬斯克創辦 Neuralink 的夥伴 Max Hodak ，在推特宣布擁有隨時都可蓋「[侏儸紀公園]」能力的報導，沒想到 Neuralink 今日就在官方[推特]上發布一段 YouTube [影片]，影片中猴子正透過自己的「意念」操控搖桿玩 MindPong！

本次實驗的設備，研究員僅透過藍芽連結到 iPhone 配對，便可直接看到植入 Pager 大腦的晶片，晶片上有將近 1024 條電極線，這些電極線會刺激猴子的神經元，不斷修正猴子在大腦中的晶片訊號，進而協調它的手臂運動。

畢竟未來 Neuralink 的終極目標，便是將這項技術運用到癱瘓的患者身上，讓患者僅需經由大腦「想像力」，便有再次行走的可能性，甚至未來能夠讓癱瘓的患者比一般人更容易操作智慧型手機。

https://buzzorange.com/techorange/2021/04/09/neuralink-monkey-learning/

**Comment by Yuhsin000** [comment]
上其他通識(腦與學習)的時候，也有提到這個技術，有看到實際操作的影片，不過設備很貴。希望以後量產造福有需求的患者!

**Comment by yuhao112** [comment]
希望這技術未來真能普及化!但依文章來看確實是還有一大段路要走呢

**Comment by sean1325418** [comment]
可能以後會跟電影一樣，每個人的大腦都埋入一個晶片

**Comment by ghost** [comment]
身為一位邊緣人，我也好希望這技術未來能夠實現並且普及，我也想交女朋友

**Comment by hapiwangy** [comment]
過去電影裡面發生的情節也已經一一在現實生活中上演了，期待有越來越多的科技可以改善我們的生活。

**Comment by ThomasMeng0515** [comment]
這或許也能成為在元宇宙中所需要的技術之一

**Comment by PoooYen** [comment]
想像一下，到時候我們真的對自己身體的自主性還剩下多少，我覺得既興奮又很可怕

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，期望腦波控制器能讓生活更加便利。

**Comment by xkllkx** [comment]
謝謝分享，但原本腦波就不是一個很乾淨、很好處理的訊號，距離普及還有一大段路要走。

**Comment by cordydeng** [comment]
這是一個不錯的開始，如果能真正應用在人們身上，想必會造福很多人

**Comment by HuanYuuuuuu** [comment]
感覺與電影中會出現的科技越來越像了!

**Comment by yenwei12138** [comment]
如果未來能普及化，肯定能造福身障殘疾等人士

**Comment by HsuChen88** [comment]
意志力就是你的超能力

**Comment by SamTU0222** [comment]
謝謝分享 認真希望這項技術可以成功的應用於身障人士身上

**Comment by handsomeniko** [comment]
好酷 謝謝分享

**Comment by Jay-722** [comment]
好酷的東西，謝謝分享

**Comment by Rosancl** [comment]
科幻電影的劇情真的一步步逐漸成真。 期待之後成功應用到身障人士身上。
